apps/settings/js/modules/dialog/alert_dialog.js

```diff
@@ -21,6 +22,15 @@ define(function(require) {
   };
 
   return function ctor_alertDialog(panelDOM, options) {
-    return new AlertDialog(panelDOM, options);
+    var d = new AlertDialog(panelDOM, options);
+    const SN_ROOT = 'body.spatial-navigation .current.' + d.DIALOG_CLASS;
+    SpatialNavigationHelper.add({
```

Comment:
please add some note here or the maintainer will likely forget why we need these code.

ex: `Support keyboard navigation in AlertDialog`